### ISSUE TITLE
Undefined timezone for a specific longitude and latitude

### ISSUE THREAD

**Issue by baneJack** [noandrea/geo2tz#22]
Hi,

No timezone for latitude:"90" and "longitude":"0"
Normally timezone must be :"Etc\GMT"

Thank you

**Comment by ext-wki** [comment]
Hi, there are more not working coordinates. The service is returning 500 error on them. I've compared with `timeshape` lib which is also based on timezone-boundary-builder and it returned the timezone.

```
curl -v http://localhost:2004/tz/43.42582/11.831443
*   Trying [::1]:2004...
* Connected to localhost (::1) port 2004
> GET /tz/43.42582/11.831443 HTTP/1.1
> Host: localhost:2004
> User-Agent: curl/8.4.0
> Accept: */*
>
< HTTP/1.1 500 Internal Server Error
< Access-Control-Allow-Origin: *
< Content-Type: application/json; charset=UTF-8
< Vary: Origin
< Date: Tue, 23 Jan 2024 06:51:40 GMT
< Content-Length: 40
<
{"message":"Error: Timezone not found"}
* Connection #0 to host localhost left intact
```

**Comment by noandrea** [comment]
@ext-wki thanks for reporting, I will push a release asap

**Comment by noandrea** [comment]
@baneJack the results comes from the `timezone-boundary-builder`, what it can be done is to add a feature to allow to define custom overrides for lat/lon coordinates, wdyt?

**Comment by noandrea** [comment]
fixed in v2.2.0
